libqtile/backend/wayland/window.py

```diff
@@ -686,10 +698,71 @@ def enable_fullscreen(self) -> None:
     def disable_fullscreen(self) -> None:
         self.fullscreen = False
 
+    def reparent(self, layer: SceneTree) -> None:
+        # already in the right layer
+        if self.tree_layer is layer:
+            return
+        self.tree_layer = layer
+        self.container.node.reparent(layer)
+
     @expose_command()
-    def bring_to_front(self) -> None:
+    def keep_below(self) -> None:
+        self.reparent(self.core.keep_below_window_tree)
+
+    @expose_command()
+    def keep_above(self) -> None:
+        self.reparent(self.core.keep_above_window_tree)
```

Comment:
In X11, we have a `kept_above/below` property. We should add that here. 

In addition, the X11 method signature includes an `enable` argument. If this is set, then `kept_above/below` is set to that value. If it's `None` then the state is toggled.